repos/spack_repo/builtin/packages/parallelio/package.py

```diff
@@ -20,6 +20,7 @@ class Parallelio(CMakePackage):
 
     license("Apache-2.0")
 
+    version("2_6_6", sha256="e32e018a521d38c9424940c7cfa7e9b1931b605f3511ee7ab3a718b69faeeb04")
```

Comment:
@jedwards4b did you move to a version with underscores intentionally? Or was this an accident?

I'd vote to keep the versions consistent if possible.